src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
```

Comment:
You don't need three lottery objects. One is enough. Also, since it is one, you can call it 3 times in a loop. 
And you don't need to call toString() explicitly, because it is called implicitly at objects in println() method